pyproject.toml

```diff
@@ -155,22 +154,28 @@ manylinux-x86_64-image = "manylinux2014"
 manylinux-aarch64-image = "manylinux2014"
 musllinux-x86_64-image = "musllinux_1_1"
 # RUNNER_OS is a GitHub Actions specific env var; define it here so it works on Cirrus CI too
-environment = {RUNNER_OS="Linux"}
+[tool.cibuildwheel.linux.environment]
+RUNNER_OS="Linux"
+PKG_CONFIG_PATH="/project/.openblas"
+LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/project/.openblas/lib"
 
 [tool.cibuildwheel.macos]
 # universal2 wheels are not supported (see gh-21233), use `delocate-fuse` if you need them
 archs = "x86_64 arm64"
 test-skip = "*_universal2:arm64"
-environment = {RUNNER_OS="macOS"}
+[tool.cibuildwheel.macos.environment]
+# In cibuildwheel, {project} is $PWD
+RUNNER_OS="macOS"
```

Comment:
I think the variables set in this section are overwritten by https://github.com/numpy/numpy/blob/main/tools/ci/cirrus_wheels.yml#L94.  THey're set in the ci config script because we have to figure out what kind of system we're building on first.

Reply:
I see. But even in a local build, with only the values fro the `pyproject.toml`, `DYLD_LIBRARY_PATH` is not percolating into the repair wheel command.

Reply:
Does the way that scipy do it help? https://github.com/scipy/scipy/blob/a6587b6c1a806d384b292592423693e4c573882c/ci/cirrus_wheels.yml#L76

Reply:
People using cibw to build locally tend to be core developers figuring out how to keep infra running.

Reply:
heh, yeah, makes sense. Who else would want wheels.